src/main/java/org/opensearch/ad/AnomalyDetectorPlugin.java

```diff
@@ -238,8 +235,8 @@ public Collection<Object> createComponents(
         */
         // AnomalyDetectionIndices is Injected for IndexAnomalyDetectorTrasnportAction constructor
         this.anomalyDetectionIndices = new AnomalyDetectionIndices(
-            client,
-            clusterService,
+            null, // client,
+            null, // clusterService,
```

Comment:
For my understanding, this will be change to SDKRestClient after the guice injection work is done for createComponent, right?

Reply:
This `Plugin` class will actually go away.  It's just here for reference until we finish all the extension point implementation.

I'm not actually sure why this has a regular constructor and not an injected one, perhaps it's the primitive int parameter.

A version of this will be created in the `AnomalyDetectorExtension` class with the correct SDK classes, and bound to itself as a component so it can be injected where needed.